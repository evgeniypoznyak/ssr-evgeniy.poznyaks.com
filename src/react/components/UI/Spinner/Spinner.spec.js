import React from 'react';
import {shallow} from 'enzyme';
import Spinner from './Spinner';

let shallowWrapper;

beforeAll(() => {
    shallowWrapper = shallow(<Spinner/>);
});

describe('Spinner', () => {
    it('should render correctly with enzyme', () => {
        expect(shallowWrapper.children.length).toBe(1);
        expect(shallowWrapper.getElement()).toMatchSnapshot();
    });
});
