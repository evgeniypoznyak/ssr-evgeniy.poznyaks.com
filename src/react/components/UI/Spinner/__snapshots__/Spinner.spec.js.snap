// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly with enzyme 1`] = `
<div>
  <ForwardRef(WithStyles)
    className="makeStyles-progress-1"
  />
</div>
`;

exports[`Spinner should render correctly with enzyme 1`] = `
<div>
  <ForwardRef(WithStyles)
    className="makeStyles-progress-1"
  />
</div>
`;
